source_audio/01-SW-042017.txt

```diff
@@ -1 +1,30 @@
\ No newline at end of file
+{
```

Comment:
Here, I removed extra beat positions temporarily to make the program faster so I can test things quickly.